indico/modules/rb_new/client/js/modules/bookRoom/BookRoomModal.jsx

```diff
@@ -197,7 +197,7 @@ class BookRoomModal extends React.Component {
     }
 
     renderRoomTimeline(availability) {
-        const hourSeries = _.range(6, 24, 2);
+        const hourSeries = _.range(6, 22, 2);
```

Comment:
What is the reason for this change?

Reply:
your comment from my previous PR:
https://github.com/indico/indico/pull/3558#discussion_r221244115